src/Microsoft.VisualStudio.SlowCheetah.Tests/Microsoft.VisualStudio.SlowCheetah.Tests.csproj

```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="MicroBuild.NonShipping" Version="2.0.54" />
+    <PackageReference Include="Microsoft.VisualStudio.Internal.MicroBuild" Version="2.0.66" />
```

Comment:
This is not an equivalent change. It *was* the `NonShipping` package and now it's not. You should still be able to find a non-shipping variant of this package under the new naming scheme.
Same for your other test projects.

Reply:
Thanks for the heads up! I fixed it in #236

Reply:
I only see one test project fixed in that PR. But in _this_ PR the bug was introduced in at least two test projects.

Reply:
Yep, I didn't notice there were 2 of them. Fixed the other one in #237